app/assets/javascripts/bastion/components/views/bst-bookmark.html

```diff
@@ -0,0 +1,44 @@
+<ul>
+  <li ng-repeat="bookmark in bookmarks"><a ng-click="get(bookmark)" class="ng-scope">{{ bookmark.name }}</a></li>
+  <li class="divider"></li>
+  <li><a ng-click="add()" translate class="ng-scope">{{ "Bookmark this search" | translate }}</a></li>
```

Comment:
You only need one or the other `translate`.  Preference to the directive version over the filter.